src/components/common/Date/Date.tsx

```diff
@@ -1,14 +1,16 @@
 import { parseISO, format } from 'date-fns';
+import classNames from 'classnames';
 import styles from './styles.module.scss';
 
 type Props = {
     dateString: string,
+    home?: boolean
```

Comment:
if it's boolean - use `is` prefix. And why we have `home` variable in generic component? Consider another name